aggsender/validator_poller.go

```diff
@@ -220,30 +230,26 @@ func (vp *validatorPoller) processResults(
 	for res := range resultsCh {
 		if res.err != nil {
 			errs = append(errs, res.err)
-
-			metrics.ValidatorError(res.validator.Address())
-
+			vp.log.Errorf("validatorRequest returned an error: %s", res.String())
```

Comment:
Does it make sense to revert the metrics?


```suggestion
            metrics.ValidatorError(res.validator.Address())
			vp.log.Errorf("validatorRequest returned an error: %s", res.String())
```

Reply:
good catch!